bot/exts/evergreen/fun.py

```diff
@@ -87,6 +88,91 @@ def conversion_func(text: str) -> str:
             converted_text = f">>> {converted_text.lstrip('> ')}"
         await ctx.send(content=converted_text, embed=embed)
 
+    @commands.group(name="caesarcipher", aliases=("caesar",))
+    async def caesarcipher_group(self, ctx: Context) -> None:
+        """
+        Translates a message using the Caesar Cipher.
+
+        See `decrpyt`, `encrypt`, and `info` subcommands.
+        """
+        if ctx.invoked_subcommand is None:
```

Comment:
I think better is use `invoke_without_command=True` in `command.group` decorator.

Reply:
I don't mind the current way, it's a fairly common pattern.

The problem with `invoke_without_command` afaik is that it allows sub-commands to bypass checks on the group cmd, which can be unexpected.

Reply:
So far I've only changed how the help embed is invoked when no subcommands are passed; I have to ask though:

> allows subcommands to bypass checks

would there be any merit to using `invoke_without_command` given that I'm only invoking the help embed for the command group?

Reply:
In this case I don't think it makes much of a difference (please correct me if I'm wrong).

Just to explain what I meant: if we use `invoke_without_command=False` (default), and apply some kind of a check, such as `with_role` from our utils, this check will also apply to *all* subcommands, which is generally what we want. This means that if you lock the group to mods only, all subcommands will also prevent non-mod users from invoking them.

With `invoke_without_command=True`, the group callback (which contains the check) will be circumvented by all subcommands, which means that they bypass the check. This can be a little surprising and unexpected.

This all is not too relevant to this case, since the group has no checks (and isn't likely to have in the future), but it's the reason why I don't like using the kwarg too much, as this behaviour is imo somewhat unexpected (although documented). So I think not having the kwarg and doing checking `ctx.invoked_subcommand` to see if the help should be invoked is perfectly fine.

But if you like @ks129's suggestion there's nothing inherently wrong with it, it will save you a line of code, since the group cmd will never run if a subcommand was invoked, it can just send the help.